ui/appui-react/src/appui-react/statusbar/popup/StatusBarPopover.tsx

```diff
@@ -0,0 +1,39 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+/** @packageDocumentation
+ * @module StatusBar
+ */
+
+import * as React from "react";
+import { Popover } from "@itwin/itwinui-react";
+import { ButtonExpandIndicator } from "./ExpandIndicator";
+
+/** Popover component used in [[StatusBar]] component.
+ * This component should wrap the element that triggers the popover.
+ * @note Add the {@link StatusBarPopover.ExpandIndicator} to popover trigger buttons.
+ * @public
+ */
+export function StatusBarPopover(props: React.ComponentProps<typeof Popover>) {
+  return (
+    <Popover
+      {...props}
+      placement="top"
+      applyBackground
+      middleware={{
+        offset: 4,
+      }}
+    />
+  );
+}
+
+/** Components used in StatusBarPopover.
+ * @public
+ */
+export namespace StatusBarPopover {
```

Comment:
Do we need a separate namespace definition? Can we assign additional properties on the function itself i.e. `StatusBarPopover.ExpandIndicator = ButtonExpandIndicator;`

Reply:
I tested it. If I write `StatusBarPopover.ExpandIndicator = ButtonExpandIndicator;` it will still export `StatusBarPopover` as an undocumented namespace so it seems like there is no way around it.